src/dosarray_run_http_experiment.sh

```diff
@@ -172,7 +172,11 @@ echo "LOG_COUNT=${LOG_COUNT} (Does this look alright?)"
 if [ ${GRAPHING} ]
 then
   echo "Running graphing"
+<<<<<<< HEAD
```

Comment:
This patch includes a "diff" -- it seems that a merge conflict hadn't been resolved before committing the file.